crates/compiler/can/src/annotation.rs

```diff
@@ -980,7 +992,12 @@ fn can_annotation_help(
                     }
 
                     None => Type::EmptyTagUnion,
-                }
+                }*/
+
+                Type::TagUnion(
+                    Default::default(),
+                    TypeExtension::from_type(ext_type, is_implicit_openness),
+                )
```

Comment:
@Anton-4 Before merging, can we clean up the diff so it's just this one and the one in the .txt file? e.g. get rid of the block comment and `dbg`s etc. 😄

Reply:
Absolutely, that's why I marked the PR as WIP